src/pages/Explorer/Category/Category.svelte

```diff
@@ -9,9 +12,49 @@
   export let iconClass = ''
   export let loading = false
   export let showLess = false
+  export let isMain = false
+
+  $: isMain && changeOrder(items)
+
+  // Formula: Likes * 0.3 + Comments * 0.7
+  const getRank = (item) => item.votes.totalVotes * 0.3 + item.commentsCount * 0.7
+
+  function changeOrder(items) {
+    items.forEach((item) => {
+      const explorerItem = getExplorerItem(item)
+      item.insertedAt = explorerItem.insertedAt
+      const insertedAt = new Date(explorerItem.insertedAt)
+      item.postdate = getPostMonthDay(insertedAt)
+      const { YYYY, MM, DD } = getDateFormats(insertedAt)
+      item.date = `${YYYY}${MM}${DD}`
+      item.rank = getRank(explorerItem)
+      item.first = false
+      item.last = false
+    })
+    items.sort((a, b) =>
+      a.postdate === b.postdate ? b.rank - a.rank : b.insertedAt - a.insertedAt,
```

Comment:
Items are returned from backend with being sorted by `insertedAt` field. Therefore `0` can be returned if the sorting is not needed
```suggestion
      a.postdate === b.postdate ? b.rank - a.rank : 0,
```